cypress/support/commands.ts

```diff
@@ -126,6 +141,38 @@ Cypress.Commands.add('mockAllRequests', () => {
   cy.mockInternalRequests()
 })
 
+// @ts-ignore
+Cypress.Commands.add('mockWebSocketRequests', (method: string, data: Object, response: Object) => {
+  let options = {
+    takeWhileFn: (message: IMessage) => message && message.method !== "end",
+    startUpMessage: {
+      method: method,
+      data: data
+    },
+  };
+  // Wrap the request in order to bypass the defaultCommandTimeout
+  // Investigating alternative solutions
```

Comment:
Can you link to https://github.com/lensesio/cypress-websocket-testing#arguments as a TODO comment instead? As I understand, we will not investigate it ourselves, but `@lensesio/cypress-websocket-testing` might get updated to override the default cypress config if that's possible.